revolv/static/js/base/dashboard.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function() {
```

Comment:
I thought we got rid of the dashboard changes, so we don't need this file, right?